tests/unit/main.cc

```diff
@@ -54,6 +54,16 @@
 #include <Kokkos_Core.hpp>
 #endif
 
+extern "C" {
+void __ubsan_on_report() {
+   FAIL() << "Encountered an undefined behavior sanitizer error";
+}
+
+void __asan_on_error() {
+  FAIL() << "Encountered an address sanitizer error";
```

Comment:
can you post an example of how does such error look in the CI? (is the complete output from ASAN included?)

Reply:
Yes. Take a look -> https://github.com/DARMA-tasking/checkpoint/pull/222/checks?check_run_id=3115530025#step:7:493

![image](https://user-images.githubusercontent.com/14062470/126351878-2c2f8e9e-34c7-4d02-ade0-9a61c4a8ddbc.png)